packages/tldraw/api/api.json

```diff
@@ -14583,7 +14583,7 @@
               "text": "export interface TLUiContextMenuProps "
             }
           ],
-          "fileUrlPath": "packages/tldraw/src/lib/ui/components/ContextMenu.tsx",
+          "fileUrlPath": "packages/tldraw/.tsbuild-api/lib/ui/components/ContextMenu.d.ts",
```

Comment:
Seems to be an api auto update too??

Reply:
I think we made some changes to how the `api.json` files look like, but we didn't apply the changes to all the existing file. Would probably be nice to create a PR and apply all the changes.